docs/lib/datastore/fragments/js/examples.md

```diff
@@ -1,6 +1,7 @@
-## Example React application
```

Comment:
I guess im confused, why is this being removed ? I do still see this fits the React application below that follows.